From f7398e8ceaa634bff73b1b1cd04ac0aa572d5249 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin@piware.de>
Date: Thu, 1 Oct 2020 15:44:43 +0200
Subject: [PATCH] Add missing "make" BuildRequires

The .spec calls make, and it's not present in the Fedora container
images.
---
 anaconda.spec.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/anaconda.spec.in b/anaconda.spec.in
index d2b34a65f..5c5146c9c 100644
--- a/anaconda.spec.in
+++ b/anaconda.spec.in
@@ -54,6 +54,7 @@ BuildRequires: gobject-introspection-devel
 BuildRequires: glade-devel
 BuildRequires: libgnomekbd-devel
 BuildRequires: libxklavier-devel >= %{libxklavierver}
+BuildRequires: make
 BuildRequires: pango-devel
 BuildRequires: python3-kickstart >= %{pykickstartver}
 BuildRequires: python3-devel
-- 
2.23.0

